.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+**/.DS_Store
```

Comment:
this should live in your global gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore